DigitalLearningSolutions.Web/Views/LearningPortal/Current/_CurrentLearningResourceCard.cshtml

```diff
@@ -0,0 +1,46 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.LearningPortal.Current
+@model LearningResourceCardViewModel
+
+<div class="searchable-element nhsuk-panel @Model.DateStyle()" id="@Model.Id-lhr-card">
+  <details class="nhsuk-details nhsuk-expander nhsuk-u-margin-bottom-0">
+    <summary class="nhsuk-details__summary nhsuk-u-padding-top-0 nhsuk-u-padding-left-0">
+      <span class="nhsuk-u-visually-hidden">@Model.DueByDescription()</span>
+      <span class="nhsuk-details__summary-text searchable-element-title" id="@Model.Id-name-lhr" name="name" data-name-for-sorting="name" role="heading" aria-level="2">
+        @Model.Name
+      </span>
+    </summary>
+    <div class="nhsuk-details__text nhsuk-u-padding-left-0">
+      <div class="nhsuk-u-padding-left-3">
+        <div class="nhsuk-grid-row">
+          <partial name="Shared/_TagRow" model="@Model" />
+        </div>
+        <div class="nhsuk-grid-row">
+          <div class="nhsuk-grid-column-full">
+            @Html.Raw(Model.ResourceDescription)
+          </div>
+        </div>
+
+        <div class="nhsuk-grid-row">
+          <partial name="Shared/_Dates" model="@Model" />
+        </div>
+      </div>
+      <a class="nhsuk-button nhsuk-button--secondary small-edit-button nhsuk-u-margin-bottom-2" aria-describedby="@Model.Id-name-lhr">
+        Mark as complete
+      </a>
+      <a class="nhsuk-button delete-button small-edit-button" aria-describedby="@Model.Id-name-lhr">
+        Remove resource
+      </a>
+      
+    </div>
+  </details>
+
+  <div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-full">
+      <a class="nhsuk-button nhsuk-u-margin-bottom-0 nhsuk-u-margin-top-0"
+         aria-describedby="@Model.Id-name-lhr"
+         href="@Model.LaunchResourceLink">
+        Launch learning resource
+      </a>
+    </div>
+  </div>

```

Comment:
I think it would be nice to have a bit more space between this button and the bottom of the card

Reply:
See above comment with pictures pasted. I've only increased it a small amount on all the cards, but I think it is acceptable now, let me know if it should be different.

Reply:
Looks good to me!